src/main/scala/me/rexim/issuestant/Main.scala

```diff
@@ -1,13 +1,32 @@
 package me.rexim.issuestant
 
+import me.rexim.issuestant.github.EventsSource
+
+import scala.concurrent.duration.{FiniteDuration, SECONDS}
+
 import scalaz.concurrent.Task
 
 import org.http4s.server.{Server, ServerApp}
 import org.http4s.server.blaze._
+import org.http4s.client.blaze._
 
 object Main extends ServerApp {
-  // TODO: Use the HTTP port passed by heroku
+  def pollingLoop(permalink: Permalink): Unit = {
+    permalink.update.flatMap { nextPermalink =>
+      Task.schedule(pollingLoop(nextPermalink), FiniteDuration(1, SECONDS))
```

Comment:
I suggest to extract that logic into another function that'll handle exceptions etc. properly. I don't like that `pollingLoop` is responsible for two things:

- a. Processing the business logic.
- b. Maintaining the async loop.

_b._ is a technical thing and could be extracted and properly tested isolated from the business logic. We're possibly going to have many "polling loops" like that for various thingies.